Projects/App/Sources/Screens/BankIDLoginScreen/Login.swift

```diff
@@ -22,3 +23,18 @@ extension Login: Presentable {
         }
     } 
 }
+
+extension MenuChild {
+    public static var login: MenuChild {
+        MenuChild(
+            title: L10n.settingsLoginRow,
+            style: .default,
+            image: hCoreUIAssets.memberCard.image
+        ) { viewController in
+            viewController.present(
+                Login(),
```

Comment:
I think modal alerts can be presented directly from the `viewController` as long as they don't contain further navigation. This will allow us to clean up navigation over time in the app - what do you think?

Reply:
@tarikstafford so what I was thinking is that the `Flow` decides which Menu it want's to provide, but as these items are fully self contained I don't think they make sense to have as a result on the individual presentable, they just launch another `Flow` on top of a flow so to say, in that way you can just provide any kind of menu to `Embark` for example.

https://github.com/HedvigInsurance/ugglan/pull/181/files#diff-3962e5870fcbdcb4d5995773f874b9607ea393366ef4ae57d2f0f9092653637fR45